libs/@guardian/source-react-components-development-kitchen/src/file-input/FileInput.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import type { ReactElement } from 'react';
+import { FileInput } from './FileInput';
+
+/**
+ * [Storybook](https://guardian.github.io/csnx/?path=/story/source-react-components-development-kitchen_fileinput--optional-file-input&globals=backgrounds.value:transparent
+ * [GitHub](https://github.com/guardian/csnx/tree/main/libs/@guardian/source-react-components-development-kitchen/src/file-input
+ * [NPM](https://www.npmjs.com/package/@guardian/source-react-components)
```

Comment:
I think this should be `.../source-react-components-development-kitchen`

Reply:
Good spot - have updated!